certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/DateRange.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.certificateservice.domain.certificate.model;
+
+import java.time.LocalDate;
+import lombok.Builder;
+import lombok.Value;
+import lombok.With;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.FieldId;
+
+@Value
+@Builder
+public class DateRange {
+
+  FieldId dateRangeId;
+  @With
```

Comment:
Can be removed? Dont see any usage of this

Reply:
Done